### ISSUE TITLE
Empty videos.txt

### ISSUE THREAD

**Issue by Rishats** [alekseylovchikov/ch-download#14]
Tested via node js: v10.15.0 and  v8.9.1
I try to download:
https://coursehunters.net/course/kak-strukturirovat-proekt-laravel
I get:
Empty videos.txt file.
<img width="950" alt="2019-02-24 22 31 16" src="https://user-images.githubusercontent.com/21194702/53414296-99695300-39f8-11e9-82fb-30891559c1df.png">

**Comment by AntonRich** [comment]
I confirm I have the issue. The issue occurs only when downloading courses that require authorization. The rest are downloading without login and password.

**Comment by a-eid** [comment]
`npm install -g https://github.com/alekseylovchikov/ch-download` and  try again.

**Comment by sldone** [comment]
same issue

**Comment by seunoyebode** [comment]
> `npm install -g https://github.com/alekseylovchikov/ch-download` and try again.

That's how i installed it and i still have the same error

**Comment by anastely** [comment]
SAME !!

**Comment by Viktor19931** [comment]
@alekseylovchikov same issue, please help to resolve this issue. Thanks

**Comment by alekseylovchikov** [comment]
@Viktor19931 are you using the latest version?

**Comment by Azghour-Saad** [comment]
@Viktor19931, Same issue, I can`t download any course it have premium , any help please, here's a screenshot, i'm  getting this :  

![Download](https://user-images.githubusercontent.com/33766249/63225547-f9cb9700-c1c8-11e9-9cac-374baff8755d.PNG)

**Comment by alekseylovchikov** [comment]
@Azghour-Saad coursehunters -u https://coursehunter.net/course/sozdayte-backend-rest-api-s-pomoshchyu-python-i-django-dlya-nachinayushchih -e example@mail.com -p password try this

**Comment by Azghour-Saad** [comment]
@alekseylovchikov , Not working, anyway ❓

**Comment by Teemitze** [comment]
@Azghour-Saad not working( Did anyone find a solution?

**Comment by Azghour-Saad** [comment]
@Teemitze, , README Updated but doesn't work, it's required paid subscription, Anyone find a solution for free please  !!! ⁉️ ⁉️

**Comment by Teemitze** [comment]
@Azghour-Saad I have a paid subscription

**Comment by Teemitze** [comment]
I could download lessons for anyone.  The offer is valid until 20/09.  Email teemitze@gmail.com

**Comment by Azghour-Saad** [comment]
@Teemitze, I can now see that it works with your paid subscription,I will contact you at this email, Thanks so much for this support !!! 👨 🥇

**Comment by Teemitze** [comment]
yes the code really works.  If I can help you with something, be sure to
write)

сб, 31 авг. 2019 г., 15:56 Azghour-Saad <notifications@github.com>:
